### PR TITLE
 jboss-forge 2.17.0.Final

### PR DESCRIPTION
**Pull request by gastaldi**

### CODE REVIEW

File: Library/Formula/jboss-forge.rb [hunk 1]
```diff
@@ -3,9 +3,9 @@
 class JbossForge < Formula
   desc "Tools to help set up and configure a project"
   homepage 'http://forge.jboss.org/'
-  url 'https://repository.jboss.org/nexus/service/local/artifact/maven/redirect?r=releases&g=org.jboss.forge&a=forge-distribution&v=2.16.2.Final&e=zip&c=offline'
-  version '2.16.2.Final'
-  sha1 'b8588f7b7bb26a0349e33c61cbb1663b6b96040c'
+  url 'https://repository.jboss.org/nexus/service/local/artifact/maven/redirect?r=releases&g=org.jboss.forge&a=forge-distribution&v=2.17.0.Final&e=zip&c=offline'
+  version '2.17.0.Final'
+  sha1 '38687342e29d0d39b36856dc5aac74b033f72ce3'
```
Comment (DomT4) [inline Library/Formula/jboss-forge.rb]: Please use `sha256` here, Thanks!

Comment (gastaldi) [inline Library/Formula/jboss-forge.rb]: Is this a new requirement? I think I missed the announcement. Will do it. Thanks

Comment (bfontaine) [inline Library/Formula/jboss-forge.rb]: Yes, we deprecated SHA1s a few weeks ago.

**Comment by dunn** [comment]
Any chance of a test as well? [CMake](https://github.com/Homebrew/homebrew/blob/master/Library/Formula/cmake.rb#L103-L106) is an example of a formula with a good test; we're just trying to check basic functionality.  Thanks!!

**Comment by gastaldi** [comment]
Sorry, but that´s beyond my Ruby knowledge :) 
I appreciate if any of you guys could help.

**Comment by dunn** [comment]
Merged in 331bc9d9464b8aa32a4d3508039a82da8f06bbab, thanks!

**Pull request closed without merging**